maas/plugins/horizon_check.py

```diff
@@ -45,8 +45,8 @@ def check(args):
     OS_USERNAME = auth_details['OS_USERNAME']
     OS_PASSWORD = auth_details['OS_PASSWORD']
     OS_USER_DOMAIN_NAME = auth_details['OS_USER_DOMAIN_NAME']
-    HORIZON_URL = 'https://{ip}'.format(ip=args.ip)
-    HORIZON_PORT = '443'
+    HORIZON_URL = 'http://{ip}'.format(ip=args.ip)
+    HORIZON_PORT = '80'
```

Comment:
It's definitely something that we will need to consider and look into. We'll need to talk closely with support to see how exactly they implement SSL. Do they terminate at the LB, or configure it in a per-service basis? Perhaps these questions can roll into some of the SPIKE work that we are doing. For now I'll reference this comment in the "Feasibility Analysis" card.